pkgs/development/python-modules/tumpa/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, pyside2
+, johnnycanencrypt
+}:
+
+buildPythonPackage rec {
+  pname = "tumpa";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "kushaldas";
+    repo = "tumpa";
+    rev = "v${version}";
+    sha256 = "1wvs64s0jxn4p8zr643d2hcczw3a175r6ib3481gdhjx38kgxjbq";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    johnnycanencrypt
+    pyside2
+  ];
+
```

Comment:
Please add a pythonImportsCheck.

Reply:
Added for johnnycanencrypt, tumpa seems to be lacking tests.

Reply:
pythonImportsCheck is a replacement for tests to know if the module can be imported at all. Please add this.